profile/src/main/java/com/scottlogic/deg/profile/reader/ConstraintReaderMapEntry.java

```diff
@@ -0,0 +1,13 @@
+package com.scottlogic.deg.profile.reader;
+
+public class ConstraintReaderMapEntry {
+    public String operatorCode;
```

Comment:
Use private fields with accessors rather than public fields. Public fields tie any usage to the implementation, allows modification from anywhere in the code and limits extensibility.